### PR TITLE
Fixes part of #4177: Dark mode Topic Activity Part 4 (Practice Tab)

### PR DESCRIPTION
**Pull request by MohitGupta121**
## Explanation

Fixes part of #4177: Dark mode Topic Activity Part 4 (Practice Tab)

Dark Mode Mockup:-

- https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/f403add1-079a-4202-9975-efe04c076290/specs/
- https://xd.adobe.com/view/c05e9343-60f6-4c11-84ac-c756b75b940f-950d/screen/33f22169-7db1-4a28-84e3-8a6cb3be2f4d/specs/

|Default|Dark Mode|
|:----------------------------------------:|:-----------------------------------------:|
|  | Screenshot_2022-05-31-12-23-27-332_com android camera |
|||
|  | Screenshot_2022-05-31-12-23-27-332_com android camera |

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@rt4914 please first merge #4571 after that merge this one, as background color that we using is already in the past PR that's why.

**Comment by MohitGupta121** [comment]
@rt4914 I use the color from last PR and make that background color common for Topics Fragments. Can you please review it now.

File: app/src/main/res/values/component_colors.xml [hunk 1]
```diff
@@ -280,6 +280,10 @@
   <color name="component_color_audio_fragment_seekbar_progress_color">@color/color_palette_icon_background_secondary_color</color>
   <color name="component_color_audio_fragment_seekbar_color">@color/color_palette_seekbar_progress_background_color</color>
   <!-- Topic Revision Fragment -->
-  <color name="component_color_topic_revision_fragment_background_color">@color/color_palette_topic_revision_background_color</color>
+  <color name="component_color_topic_revision_fragment_background_color">@color/color_palette_topic_fragment_background_color</color>
+  <!-- Topic Practice Fragment -->
+  <color name="component_color_topic_practice_background_color">@color/color_palette_topic_fragment_background_color</color>
```
Comment (rt4914) [inline app/src/main/res/values/component_colors.xml]: Replace `topic_practice` with `topic_practice_fragment` in all three values.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@MohitGupta121 PTAL, Thanks.

File: app/src/main/res/values/component_colors.xml (hunk 1 above)
Comment (MohitGupta121) [reply app/src/main/res/values/component_colors.xml]: Okay, actually the background color in palette is common for whole topic that's why I make it common.
But I'll change `topic_practice` with `topic_practice_fragment` in components color.

**Comment by MohitGupta121** [comment]
Done @rt4914 can you please now review it.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

**Pull request merged**